pkg/cmd/cluster/cluster_open.go

```diff
@@ -0,0 +1,23 @@
+package cluster
+
+import (
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+// newCmdClusterOpen opens the current cluster UI in the user browser.
+func newCmdClusterOpen(ctx api.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "open",
+		Short: "Open the current cluster UI in the browser",
+		Args:  cobra.NoArgs,
```

Comment:
We could also accept an optional argument here, eg. `dcos cluster open soak114`

Reply:
Yes I think it would make sense to have this option.